server/auth_server.go

```diff
@@ -39,11 +39,13 @@ func (a *AuthServer) IsWorkspaceAuthenticated(ctx context.Context, request *api.
 	}
 	workspaceAndNamespace := xOriginalAuth[0:pos]
 	pieces := strings.Split(workspaceAndNamespace, "--")
-	client := ctx.Value("kubeClient").(*v1.Client)
 	allowed, err := auth.IsAuthorized(client, pieces[1], "create", "apps/v1", "statefulsets", pieces[0])
-	if err != nil || !allowed {
+	if err != nil {
 		return &empty.Empty{}, err
 	}
+	if !allowed {
```

Comment:
No need for this now since this is addressed in this [PR](https://github.com/onepanelio/core/pull/174/files#diff-b49d465c8dbf2c888fe2e2bd0c18e167R81)

Reply:
Made change.